backend/api/Program.cs

```diff
@@ -69,7 +69,7 @@
 var app = builder.Build();
 
 // Configure the HTTP request pipeline.
-if (app.Environment.IsDevelopment())
+if (app.Environment.IsDevelopment() || app.Environment.IsStaging())
```

Comment:
Maybe this should be for all environments? I am assuming we want swagger for documentation in Production as well

Reply:
I think we will change this part slightly as we evolve. There will probably be separate sections for each environment so I would say we just add it when we need it.